packages/react-slider/src/components/RangedSlider/useRangedSliderStyles.ts

```diff
@@ -0,0 +1,168 @@
+import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
+import {
+  thumbClassName,
+  trackClassName,
+  useRootStyles,
+  useSliderWrapper,
+  useRailStyles,
+  useTrackWrapperStyles,
+  useTrackStyles,
+  useMarksWrapperStyles,
+  useThumbWrapperStyles,
+  useThumbStyles,
+  useActiveRailStyles,
```

Comment:
nit: Order alphabetically
```suggestion
  thumbClassName,
  trackClassName,
  useActiveRailStyles,
  useMarksWrapperStyles,
  useRailStyles,
  useRootStyles,
  useSliderWrapper,
  useThumbStyles,
  useThumbWrapperStyles,
  useTrackStyles,
  useTrackWrapperStyles,
```